core/platform/models.py

```diff
@@ -191,8 +190,8 @@ def get_all_storage_model_classes(cls):
             list(class). The corresponding storage-layer model classes.
         """
         model_names = [
-            name for name in NAMES.__dict__
-            if '__' not in name and name != 'base_model']
+            name for name in NAMES
+            if '__' not in name.value and name.value != 'base_model']
```

Comment:
Again `if '__' not in name.value` might not be needed (see comment below for details).

Reply:
done